### ISSUE TITLE
Distinguish between bulk policies on front page

### ISSUE THREAD

**Issue by lgarron** [chromium/hstspreload.org#106]
Follow-up to recent changes, as noticed in https://github.com/EFForg/https-everywhere/issues/13147

**Comment by vcsjones** [comment]
This seems like a good idea. The idea being that we could flag certain errors as likely to lead to removal and adjust the UI accordingly, correct?

While I tinker with TypeScript, anything I can do to help here?

**Comment by lgarron** [comment]
> This seems like a good idea. The idea being that we could flag certain errors as likely to lead to removal and adjust the UI accordingly, correct?

Yeah, right now we apply the new 1-year calculation to all old entries (which only have an 18-week requirement): https://github.com/EFForg/https-everywhere/issues/13147#issuecomment-336672380

A short-term solution would be:
- Modify `chromium/hstspreload` to support modifying calculations (or filtering errors) based on a policy enum.
- Apply the `bulk-18-weeks` policy to all domains currently marked as bulk on the server.

In the longer term, we'll need to support at least three different policies (old bulk, 18-week bulk, and 1-year bulk), which is best done by annotating the preload list source. I plan to work on that soon.
